README.md

```diff
@@ -109,6 +109,15 @@ END_COMMIT_OVERRIDE
 The next time release please runs, it will use that override section as the
 commit message instead of the merged commit message.
 
+## Release Please bot does not create a release PR. Why?
+
+Release Please creates a release pull request after it sees the default branch
+contains "releaseable units" since the last release.
+A releasable unit is a commit to the branch with one of the following
+prefixes: "feat" and "fix".
```

Comment:
@Neenu1995 There may be more types of releasable unit prefixes but I couldn't find a corresponding document.